enterprise/suppress-user/factory_test.go

```diff
@@ -0,0 +1,141 @@
+package suppression
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/gorilla/mux"
```

Comment:
Should we use `chi` instead?

Reply:
although, given than we haven't introduced `chi` yet in rudder-server. I believe it is fine for now. I will take it up as a separate task for moving from `mux` to `chi` in rudder-server.

Reply:
Yeah sure :+1: